NEWS.md

```diff
@@ -1,4 +1,10 @@
-# pins (development version)
+# pins (development version, to be released as 1.4.0)
+
+## Breaking changes
```

Comment:
I've started calling these sections "Lifecycle changes" which is a bit less harsh than breaking, but still gets the point across.